Contribute/Style-Guide/index.md

```diff
@@ -39,59 +58,81 @@ Try not to use any of the mentioned words, as they are often opinionated. What m
 
-1. `backoffice`
-1. `back office`
-1. `back-office`
+In order to ensure that markup languages and other names are capitalized correctly a rule has been added to check for this.
 
-This rule will kick in if you spell it as number 2 or 3 and suggest you spell it as number 1. For a full list of terms please check the [style rule](https://github.com/umbraco/UmbracoDocs/blob/master/.github/styles/UmbracoDocs/Terms.yml).
+The rule will ensure that instances of *HTML* and *CSS* are always written using only capital letters. It will also check whether *JavaScript* is written in full or if the abbreviation, *js*.
+
+### Brands
+
+In some cases throughout our documentation we refer to other software providers or brands. We have added a rule to ensure that the most commonly used brand names are spelled and capitalized correctly.
+
+The rule will, as an example, ensure that the names *Microsoft* and *Slack* are always capitalized.
 
 ## Using Vale locally
 
-One of the big strengths of Vale is that it is possible as a contributor to run the tests locally before you create a PR, there is also a VS Code extension for it.
+One of the big strengths of Vale is that it is possible as a contributor to run the tests locally before you create a PR.
 
 First step to running Vale locally is to install it following Vale's [Installation documentation](https://vale.sh/docs/vale-cli/installation/).
 
-Next you can open a command line tool in the documentation repo and run the following command:
+Next you can open a command line tool in the documentation repository and run the following command:
 
 ```vale --glob='*.md' .```
 
 This tells Vale to test all markdown files (.md) in the current directory (.). The output will look something like this:
 
 ![Vale Output](images/vale-output.png)
 
-So it tells you what file has issues, in this case `v8documentation.md`, it broke the [HeadingsPunctuation rule](#punctuation-in-headings), and it did so the following places:
+It will show you what file has issues. In the case above the `v8documentation.md` article broke the [HeadingsPunctuation rule](#punctuation-in-headings), and it did so the following places:
 
 - Line 15, column 36
 - Line 59, column 15
 - Line 64, column 12
 
-At the end of the list of errors it will tell you the total amount of errors and warnings and how many files it went through.
+When the check has run you will get the total amount of errors, warnings and suggestions including how many files has been checked.
 
-### VS Code extension
+## Visual Studio Code extension
 
-Taking it even further there is an extension for Visual Studio Code that allows you to use Vale as you are writing documentation. It is called `vale-vscode` and can be downloaded via the VS Code Marketplace in your editor.
+There is an extension for Visual Studio Code hiwch allows you to use Vale as you are writing documentation. It can also be used to run checks on existing articles and find where the potential changes are needed.
```

Comment:
```suggestion
There is an extension for Visual Studio Code which allows you to use Vale as you are writing documentation. It can also be used to run checks on existing articles and find where potential changes are needed.
```